File: PN6.java
//DisplayBackward

import java.lang.*;
import java.util.*;

class Number
{
	private int iNo;
	
	public void Accept()
	{
		
	  Scanner sobj = new Scanner (System.in);
	  System.out.println("Enter number : ");
	  this.iNo = sobj.nextInt();
	}
	
	public void DisplayBackward()
	{
		int iCnt = 0;
		
		for(iCnt = iNo;iCnt >=1;iCnt--)
		{
			System.out.println(iCnt);
		}
	}
}


class PN6
{
	public static void main(String y[])
	{
		Number nobj = new Number();
		
		nobj.Accept();
		
		nobj.DisplayBackward();
	}
}

File: PN6.cpp


#include<iostream>
using namespace std;



class Number
{
    private:
        int iNo;        

    public:
        
        void Accept()      
        {
            cout<<"Enter the value : "<<endl;
            cin>>this->iNo;
        }
		
		bool Display()
		{
		    if(
              ((iNo % 3) == 0) &&
              ((iNo % 5) == 0)
               )
               {
                  return true;
               }
               else
               {
                  return false;
               }
		}

       
};

int main()
{
    Number nobj;
    bool bRet = false;
	
    nobj.Accept();
    bRet = nobj.Display();

    if(bRet = true)
	{
		cout<<"The number is divisible by 3 & 5"<<endl; 
	}
	else
	{
		cout<<"The number is not divisible by 3 & 5 "<<endl;
	}

    return 0;
}